pages/category/[slug].tsx

```diff
@@ -0,0 +1,44 @@
+import type { FunctionComponent } from 'react';
+import type { Story } from '@prezly/sdk';
+import { GetServerSideProps } from 'next';
+import { getPrezlyApi } from '@/utils/prezly';
+import Layout from '@/components/Layout';
+import Stories from '@/modules/Stories';
+import { Category } from '@prezly/sdk/dist/types';
+
+type Props = {
+    stories: Story[];
```

Comment:
Sometimes we have `Type[]` and sometimes `Array<Type>`. We should also establish which approach we should use, for having it consistent everywhere